samples/wifi/shell/README.rst

```diff
@@ -71,7 +71,26 @@ Supported CLI commands
    * - Subcommands
      - Description
    * - scan
-     - Scan for Wi-Fi APs
+     - | Scan for Wi-Fi APs
+       | OPTIONS:
+       | [-t, --type <active / passive>] : Preferred mode of scan. The actual mode
+       | of scan can depend on factors such as the Wi-Fi chip implementation,
+       | regulatory domain restrictions etc. Default type is active.
+       | [-b, --bands <Comma separated list of band values (2/5/6)>] : Bands to be
+       | scanned where 2: 2.4 GHz, 5: 5 GHz, 6: 6 GHz.
+       | [-a, --dwell_time_active <val_in_ms>] : Active scan dwell time (in ms) on
+       | a channel. Range 5 - 1000 ms.
+       | [-p, --dwell_time_passive <val_in_ms>] : Passive scan dwell time (in ms)
+       | on a channel. Range 10 - 1000 ms.
+       | [-s, --ssid <Comma separate list of SSIDs>] : SSID list to scan for.
+       | [-m, --max_bss <val>] : Max BSSIDs to scan for. Range 1 - 65535.
+       | [-c, --chan <Comma separated list of channel ranges>] : Channels to be
+       | scanned. The channels have to be specified in the form
+       | band1:chan1,chan2_band2:chan3,..etc. The chan has to be a list of comma
```

Comment:
```suggestion
       | band1:chan1,chan2_band2:chan3,..etc. The channel has to be a list of comma
```

Reply:
This was intended to refer to chan1, chan2 etc. Updated the comment accordingly.